leb128/__init__.py

```diff
@@ -10,14 +10,24 @@
 import typing
 
 
+class Leb128Error(Exception):
+    pass
+
+
 class _U:
+    """Unsigned LEB128 codec."""
+
     @staticmethod
     def encode(i: int) -> bytearray:
         """Encode the int i using unsigned leb128 and return the encoded bytearray."""
-        assert i >= 0
+
+        if i < 0:
+            msg = f"{i} < 0"
+            raise Leb128Error(msg)
```

Comment:
https://stackoverflow.com/questions/13534563/is-it-bad-practice-to-use-assert-to-verify-assumptions-about-arguments-in-python